src/OAuthClient/ItemClient.php

```diff
@@ -0,0 +1,32 @@
+<?php
+namespace NYPL\HoldRequestResultConsumer\OAuthClient;
+
+
+use NYPL\HoldRequestResultConsumer\Model\DataModel\Item;
+use NYPL\Starter\APILogger;
+use NYPL\Starter\Config;
+
+class ItemClient extends APIClient
+{
+    public static function getItemByIdAndSource($itemId = '', $nyplSource)
+    {
+        $url = Config::get('API_BASE_URL') . '/items/' . $nyplSource . '/' . $itemId;
```

Comment:
Perhaps it would be better to set the entire string as an environment parameter, like API_ITEM_URL, instead of setting the entire string here?